tests/workbenches/notebook_images/utils.py

```
@@ -0,0 +1,781 @@
+                            "imagePullPolicy": "Always",
+                            "livenessProbe": probe_config,
+                            "name": name,
+                            "ports": [{"containerPort": 8888, "name": "notebook-port", "protocol": "TCP"}],
+                            "readinessProbe": probe_config,
+                            "resources": {
+                                "limits": {"cpu": "2", "memory": "4Gi"},
+                                "requests": {"cpu": "1", "memory": "1Gi"},
+                            },
+                            "volumeMounts": [
+                                {"mountPath": "/opt/app-root/src", "name": name},
+                                {"mountPath": "/dev/shm", "name": "shm"},
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)